functions/index.js

```diff
@@ -44,7 +44,7 @@ exports.dailycc = functions.https.onRequest(() => {
 	};
 });
 
-exports.affirmations = functions.https.onRequest((req, res) => {
+exports.affirmations = functions.https.onRequest((request, response) => {
```

Comment:
Maybe the affirmations logic should be its own module too. Wdyt?

Reply:
I agree. I like the way @jonathan-elize set it up